ts/Core/Tooltip.ts

```diff
@@ -145,7 +145,7 @@ class Tooltip {
 
     public now: Record<string, number> = {};
 
-    public options: TooltipOptions = {};
+    public options: TooltipOptions = {} as any;
```

Comment:
Can we set it here with the defaults? Should the defaults be moved to the class maybe?

Reply:
Yes probably, but I didn't want to make too many structural changes in this PR.